bin/smoke-test.sh

```diff
@@ -592,13 +620,14 @@ while getopts "h?t:v:p:r:u:s:fxlLCcd2" opt; do
             echo "  -l         Disable location-trusted work-around"
             echo "  -L         Enable location-trusted work-around"
             echo "  -2         Force client not to use TLS v1.3 as work-around"
-            echo "  -t <token> Use <token> for non-X.509 authn to target"
+            echo "  -t <token> Use <token> for non-X.509 authn to target (valid values: macaroon, scitoken, wlcg)"
```

Comment:
I'm wondering whether `wlcg` is the best choice of name here.

From what I can see, the added support is more general that this.  Any token returned by oidc-agent is used (which might or might not be an WLCG-AuthZ-WG-Profile).  If there's any requirement, it's that the token must be a JWT.

Perhaps `oidc-agent` instead of `wlcg`?